.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ['3.6', '3.7', '3.8']
+        toxenv: ['py']
+        include:
```

Comment:
you can just add `quality` in `toxenv` array, `toxenv: ['py', 'quality']`

Reply:
I'll have to exclude both `python3.6` and `python3.7` in that case to only run quality with `python3.8`.